app/src/main/java/com/doubtless/doubtless/screens/dashboard/usecases/FetchUserDataUseCase.kt

```diff
@@ -2,25 +2,24 @@ package com.doubtless.doubtless.screens.dashboard.usecases
 
 import android.util.Log
 import androidx.annotation.WorkerThread
-import com.doubtless.doubtless.DoubtlessApp
 import com.doubtless.doubtless.constants.FirestoreCollection
 import com.doubtless.doubtless.screens.auth.User
+import com.doubtless.doubtless.screens.auth.UserAttributes
 import com.doubtless.doubtless.screens.doubt.DoubtData
 import com.google.firebase.firestore.AggregateSource
 import com.google.firebase.firestore.FirebaseFirestore
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.async
+import kotlinx.coroutines.tasks.await
 import kotlinx.coroutines.withContext
 import java.util.concurrent.CountDownLatch
 import java.util.concurrent.TimeUnit
 
 class FetchUserDataUseCase constructor(
```

Comment:
you are also fetching feed in this usecase, hence change the name to FetchUserProfileFeedUseCase